lib/datadog/tracing/contrib/http/distributed/b3.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+# typed: true
+
+require_relative '../../../distributed/b3'
+require_relative 'fetcher'
+
+module Datadog
+  module Tracing
+    module Contrib
+      module HTTP
+        module Distributed
+          # B3-style trace propagation through HTTP headers.
+          # @see https://github.com/openzipkin/b3-propagation#multiple-headers
+          class B3 < Tracing::Distributed::B3
+            def initialize
+              super(fetcher: Fetcher)
+            end
+          end
+        end
```

Comment:
Should this/b3_single/datadog be removed after the refactor where propagation directly instances the superclass with the fetcher?

Reply:
👍